README.md

```diff
@@ -1,38 +1,50 @@
-LevelDOWN
+leveldown
 =========
 
-<img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
-
-**A Low-level Node.js LevelDB binding**
+> A low level node.js LevelDB binding
```

Comment:
- Why lowercase Node.js?
- low level => low-level (I think)

Reply:
No reason, other than I think it looks bad. Updating.